flocker/node/_docker.py

```diff
@@ -29,6 +30,9 @@
     RestartNever, RestartAlways, RestartOnFailure, pset_field, pvector_field)
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Since the problem only appears on Buildbot, I had the logging going to stderr, since that was the only way to see the output.  Should I revert the changes to keep that logging?  It adds a lot of noise to the test output, but not sure where else it could go, and still be visible on Buildbot.